admin/commands/management/add_proxysql.go

```diff
@@ -112,31 +106,36 @@ func (cmd *AddProxySQLCommand) RunCmd() (commands.Result, error) {
 		}
 	}
 
+	if cmd.CredentialsSource == "" {
+		if cmd.Username == "" {
+			cmd.Username = cmd.GetDefaultUsername()
+		}
+
+		if cmd.Password == "" {
+			cmd.Password = cmd.GetDefaultPassword()
+		}
+	}
+
```

Comment:
Is there a specific reason we replace an empty password or username if not provided?
I could imagine if both are empty, we apply defaults, but password could be an empty string, right?

Reply:
Default credentials for both username and password were "admin" (line 52 - 53), so if `--credentials-source` parameter wasn't passed and `--password` also, pmm-admin should apply 'admin' as default.

Reply:
That's correct. There was one case though which we don't support now.

```
pmm-admin add --username=root --password=""
```

This allows you to configure an empty password while the new solution does support this.
That is, of course, unless we did not discover a bug in Kong where the command would use a default value instead :)

Reply:
Got your point, thank you.

If we run pmm-admin from main branch: 
```
pmm-admin add proxysql --username=root --password=""
```
pmm-server will get username=root and password="" in request.
For now pmm-admin allows sending empty password even if default value for proxysql is `admin`.

In this PR even if we pass or not empty parmeter `--password=""` the pmm-server will get default `admin` password in request from admin. 


![2022-09-13_16-31](https://user-images.githubusercontent.com/12951615/189937115-8cbba518-5e63-4270-b866-3271f795e622.png)
![2022-09-13_16-41](https://user-images.githubusercontent.com/12951615/189937127-a8522030-8834-4aff-85e2-6982d8e7580f.png)
![2022-09-13_16-44](https://user-images.githubusercontent.com/12951615/189937130-e30f5984-77d1-42fa-83e7-2a9f5b7feb6c.png)

That's interesting case.
Should pmm-admin support passing empty parameters/passwords even if defaults values has been declared?
CC @BupycHuk

Reply:
> Yeah, it should support it, another question is can we differentiate the flag which wasn't passed from the flag which is empty for reason?

Unfortunately, I don't have idea how we can distinguish them :disappointed:

Reply:
It appears we shall be able to distinguish this through [Value.Set](https://pkg.go.dev/github.com/alecthomas/kong#Value) attribute.

Reply:
Fixed. 
![image](https://user-images.githubusercontent.com/12951615/197246311-1ff87235-ea50-42e5-a393-23d189958c82.png)